aten/src/ATen/native/native_functions.yaml

```diff
@@ -3371,7 +3371,7 @@
   dispatch:
     CUDA: _cslt_compress
 
-- func: _cslt_sparse_mm(Tensor compressed_A, Tensor dense_B, Tensor? bias=None, Tensor? alpha=None, ScalarType? out_dtype=None, bool transpose_result=False, int alg_id=0, int split_k=1, bool split_k_one_kernel=True) -> Tensor
```

Comment:
Can we add this as a new kwarg instead of rename the existing one? Then we can throw a warning when `split_k_one_kernel` is used. That way we can mantain BC and deprecate the `split_k_one_kernel` in the subsequent version.

Reply:
Hi @jcaip Could you elaborate on the BC change? is _cslt_sparse_mm a public API? 
If new kwargs have to be added, I'd suggest also add `split_k_buffers` (see https://docs.nvidia.com/cuda/cusparselt/types.html#cusparseltmatmulalgattribute-t) because it's also part of the split-k parameter that's tuned by the search routine.

Reply:
@j4yan It's a private API, I'm just suggesting a way that this change could be made safer. 

Anything that calls `torch._cslt_sparse_mm(... , split_k_one_kernel=True)` will break with this change. But as I said aboveI feel like this is pretty low risk so if there's a pressing need for this I'm okay with approving to unblock.